src/Microsoft.DotNet.XHarness.iOS.Shared/Execution/Mlaunch/Arguments.cs

```diff
@@ -318,4 +318,21 @@ public VerbosityArgument()
 
         public override string AsCommandLineArgument() => "-v";
     }
+
+    /// <summary>
+    /// Create a tcp tunnel with the iOS device from the host.
+    /// </summary>
+    public sealed class TcpTunnelArgument : MlaunchArgument
+    {
+        readonly int port;
+
+        public TcpTunnelArgument(int port)
+        {
+            if (port <= 0)
```

Comment:
impossible for this to be null or > 65535?

Reply:
int cannot be null, if is too large, the exception will be raised later